packages/library-base/src/nativeDarwin/kotlin/io/realm/kotlin/internal/platform/SystemUtils.kt

```diff
@@ -57,16 +57,16 @@ public actual fun epochInSeconds(): Long =
 @Suppress("MagicNumber")
 internal actual fun currentTime(): RealmInstant {
     val secs = NSDate().timeIntervalSince1970
```

Comment:
Please add the type here, it looks to be Double. It was a bit hard to reason about the below stuff without that information.